app/scripts/views/app.js

```diff
@@ -69,9 +69,9 @@ define([
         if (currentView) {
           if (currentView instanceof View) {
             // if the View to display is the same as the current view, then
-            // the user is navigating from a subview back to the parent view.
+            // the user is navigating from a childView back to the parent view.
             // No need to re-render, but notify interested parties of the event.
-            self._notifications.trigger('navigate-from-subview', options);
+            self._notifications.trigger('navigate-from-childView', options);
```

Comment:
Hmmm, not sure how I feel about mixing camel and hypens. If something is hyphened it should be all hyphened, no?